core/block_validator.go

```diff
@@ -137,6 +140,52 @@ func (v *BlockValidator) ValidateBody(block *types.Block) error {
 	return nil
 }
 
+// ValidateSystemTxs validates all system txs contained in a block.
+// We check that:
```

Comment:
Do we need to ensure that system transactions are in a contiguous block before or after the L1MessageTxs?

I think this might have implications to how we process these in the rollup contract, if they're in a predefined block then it's enough to provide a list, we don't need to attach information about the tx's position inside the block..

Reply:
Hrm, not sure it matters. Maybe we should require them to be after L1Messages.

Reply:
updated to have the first, in a contiguous "chunk" and l1 txs following.